### PR TITLE
Complete website documentation for scanners

### LINKED ISSUE #3601: Add Website Documentation for Scanners
## Story
Resource Scanner and Role Scanner are not documented on our website.  Please add these and look through other scanners that also need to be added.

https://forsetisecurity.org/docs/latest/configure/scanner/descriptions.html

## Solution
- Descriptions are added on the website for resource scanner and role scanner.
- Look through [all the scanners that we have](https://github.com/forseti-security/forseti-security/tree/master/google/cloud/forseti/scanner/scanners), and ensure there are corresponding descriptions on the website.

### PR DESCRIPTION
**Pull request by red2k18**
Resolves #3601 
- [x] Added documentation for 12 scanners.

### CODE REVIEW

File: _docs/_latest/configure/scanner/default-rules.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ resources.
 
 ---
 
+## Audit logging
```
Comment (gkowalski-google) [inline _docs/_latest/configure/scanner/default-rules.md]: I don't think there are any default rules for audit logging. This scanner is disabled by default and there are [no default rules](https://github.com/forseti-security/terraform-google-forseti/blob/master/modules/rules/templates/rules/audit_logging_rules.yaml).

File: _docs/_latest/configure/scanner/default-rules.md [hunk 2]
```diff
@@ -36,29 +41,73 @@ resources.
   * Forbid any Cloud IAP bypasses on all resources in my organization, when Cloud IAP is enabled.
   * Allow direct access from debug IPs and internal monitoring hosts.
 
+## Enabled APIs
```
Comment (gkowalski-google) [inline _docs/_latest/configure/scanner/default-rules.md]: Same as audit_logging, no default rules.

File: _docs/_latest/configure/scanner/default-rules.md [hunk 3]
```diff
@@ -36,29 +41,73 @@ resources.
...
+  * Only supported APIs should be enabled for all projects.
+  
 ## External Project Access
   * Find any users in your org that may have access to projects outside of your allowed org or folder.
   
 ## Firewall
   * Prevent allow all ingress (used to detect allow ingress to all policies)
 
-## G Suite
+## Forwarding
```
Comment (gkowalski-google) [inline _docs/_latest/configure/scanner/default-rules.md]: No default rules

File: _docs/_latest/configure/scanner/default-rules.md [hunk 4]
```diff
@@ -36,29 +41,73 @@ resources.
...
   
+## Group Settings
+  * Only allow the following supported settings for groups with IAM policies:
+    * A user from outside of the organization can never join the group.
+    * Invitation is required to join.
+    * Managers can invite people to join the group.
+    * Managers can add people to the group.
+    * Managers can leave the group.
+  
+## Instance Network Interface
```
Comment (gkowalski-google) [inline _docs/_latest/configure/scanner/default-rules.md]: No default rules.

File: _docs/_latest/configure/scanner/default-rules.md [hunk 5]
```diff
@@ -36,29 +41,73 @@ resources.
...
+  * Logging should be enabled on the projects.
     
 ## Kubernetes Engine Version
   * Only allow the following supported versions:
     * For major version 1.8, the minor version must be at least 12-gke.1
     * For major version 1.9, the minor version must be at least 7-gke.1
     * For major version 1.10, the minor version must be at least 2-gke.1
     * For major version 1.11, any minor version is allowed
+    
+## Lien
```
Comment (gkowalski-google) [inline _docs/_latest/configure/scanner/default-rules.md]: No default rules.

File: _docs/_latest/configure/scanner/default-rules.md [hunk 6]
```diff
@@ -36,29 +41,73 @@ resources.
...
+  * All projects in the specified organization should have liens to block
+    the project's deletion.
+    
+## Location
```
Comment (gkowalski-google) [inline _docs/_latest/configure/scanner/default-rules.md]: No default rules.

File: _docs/_latest/configure/scanner/default-rules.md [hunk 7]
```diff
@@ -36,29 +41,73 @@ resources.
...
+  * Buckets in the specified organization must be in the US.
+  * Buckets in the specified organization must not be in Europe.       
+  
+## Log Sink
```
Comment (gkowalski-google) [inline _docs/_latest/configure/scanner/default-rules.md]: No default rules.

File: _docs/_latest/configure/scanner/default-rules.md [hunk 8]
```diff
@@ -36,29 +41,73 @@ resources.
...
+  * Ensures all projects have specified log sinks.
+  * Ensures destination and filter specified matches that of log sink's. 
+  
+## Resource
```
Comment (gkowalski-google) [inline _docs/_latest/configure/scanner/default-rules.md]: No default rules.

File: _docs/_latest/configure/scanner/default-rules.md [hunk 9]
```diff
@@ -36,29 +41,73 @@ resources.
...
+  * Resource trees should have a match in the Forseti inventory.
+
+## Retention
```
Comment (gkowalski-google) [inline _docs/_latest/configure/scanner/default-rules.md]: No default rules.

File: _docs/_latest/configure/scanner/default-rules.md [hunk 10]
```diff
@@ -36,29 +41,73 @@ resources.
...
+  * All buckets and big tables in the organization should be deleted or 
+    overwritten in the retention period specified.
+  
+## Role
```
Comment (gkowalski-google) [inline _docs/_latest/configure/scanner/default-rules.md]: No default rules.

File: _docs/_latest/configure/scanner/descriptions.md [hunk 11]
```diff
@@ -32,7 +32,7 @@ all of the VM instances in your environment and determines if any VMs
 with external IP addresses are on a specific blacklist you've configured.
 
 For examples of how to define scanner rules, see the
-[`blacklist_rules.yaml`](https://github.com/forseti-security/forseti-security/blob/master/samples/scanner/scanners/blacklist_rules.yaml) rule file.
+[`blacklist_rules.yaml`](https://github.com/forseti-security/terraform-google-forseti/blob/master/modules/rules/templates/rules/blacklist_rules.yaml) rule file.
```
Comment (gkowalski-google) [inline _docs/_latest/configure/scanner/descriptions.md]: I'm curious why the rules paths were changed. The default rules were recently moved to the samples directory of the FS repo. I think these links should remain pointing to the sample dir.

Comment (red2k18) [reply _docs/_latest/configure/scanner/descriptions.md]: Updated links to point to FS repo samples dir.

File: _docs/_latest/configure/scanner/default-rules.md [hunk 12]
```diff
@@ -35,23 +35,34 @@ resources.
...
-    algorithm: GOOGLE_SYMMETRIC_ENCRYPTION
-    protection_level: SOFTWARE
-    purpose: ENCRYPT_DECRYPT
-    state: ENABLED
+    * algorithm: GOOGLE_SYMMETRIC_ENCRYPTION
+    * protection_level: SOFTWARE
+    * purpose: ENCRYPT_DECRYPT
+    * state: ENABLED
+    
+## Kubernetes Engine
```
Comment (gkowalski-google) [inline _docs/_latest/configure/scanner/default-rules.md]: This scanner is not enabled by default.

Comment (red2k18) [reply _docs/_latest/configure/scanner/default-rules.md]: Removed.

**Pull request merged**